---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

----------------

EDUCATION
======
* **PhD** Computer Science and Informatics (Concentration: Biomedical Informatics) - Aug 2025 (Expected)
* **MSc** Computer Science and Informatics (Interim Masters) - May 2024
* **BSc** Biomedical Engineering, Georgia Institute of Technology - Dec 2020

RESEARCH
======
**Graduate Researcher, Bhasin Lab** - Emory University (2021-), Atlanta, GA
* Developing CAR-T target identification platform using single-cell RNA sequencing data from leukemia and vital tissues. Identifying targets or pairs of targets, including for logic-gated CARs
* Utilized supervised learning methods to predict protein expression from mRNA measurements by training with paired CITE-seq data, addressing poor correlation between ADT and RNA levels to accurately label distinct blast populations in MPAL
* Performed pediatric Pan-Leukemia transcriptomic analysis to identify 9 genes dysregulated in leukemia and over-expressed at diagnosis in AML, ALL, and MPAL
* Developed [PedSCAtlas](https://bhasinlab.bmi.emory.edu/PediatricSCAtlas/), interactive tool for analyzing compiled datasets of pediatric single-cell RNA-seq cancer datasets for pediatric cancer subtypes
* Performed single-cell RNA-seq analysis of pediatric mixed phenotype acute leukemia (MPAL), and comparison to other acute leukemia subtypes
* Validated our groupâ€™s 7-gene signature for identification of AML malignant blast cells using external single-cell and bulk RNA-seq datasets
* Assisted with various bioinformatics projects in the lab, such as the development and maintenance of the web resource Survival Genie, single-cell analysis of T-cell acute lymphoid leukemia immune microenvironment, and the single-cell analysis of diabetic foot ulcer healing

**Undergraduate Researcher, Jo Lab** - Georgia Institute of Technology (2019-2020), Atlanta, GA
* Performed scRNA-seq, scATAC-seq, and lncRNA analysis to study endothelial cell reprogramming in the carotid artery

**Data Analysis Internship** - Open Medicine Institute (2018), Mountain View, CA
* Developed a single-cell RNA sequencing protocol for their patient data
* Developed and implemented personalized healthcare solutions
  
PUBLICATIONS
======
* **Mumme HL**, et al. Identification of leukemia-enriched signature through the development of a
comprehensive pediatric single-cell atlas. ***Nat Commun*** 16, 4114 (2025).
[https://doi.org/10.1038/s41467-025-59362-5](https://doi.org/10.1038/s41467-025-59362-5)
* Yadalam AK, Chaudagar K, **Mumme H**, et. al. Rapid modulation of circulating adipokines and
inflammatory cytokines in localized prostate cancer following short-term leuprolide therapy. ***Clinical
Genitourinary Cancer*** 23, 4 (2025). [https://doi.org/10.1016/j.clgc.2025.102358](https://doi.org/10.1016/j.clgc.2025.102358)
* Bakhtiari M, Jordan SC, **Mumme HL**, et al. ARMH1 is a novel marker associated with poor pediatric AML outcomes that affect the fatty acid synthesis and cell cycle pathways. ***Frontiers in Oncology*** (2024). [https://doi.org/10.3389/fonc.2024.1445173](https://doi.org/10.3389/fonc.2024.1445173)
* Choi D, Bakhtiari M, Pilcher W, Huang C, Thomas BE, **Mumme H**, et al. Single-cell analysis of debrided diabetic foot ulcers reveals dysregulated wound healing environment in non-hispanic blacks. ***JID*** (2024). [https://doi.org/10.1016/j.jid.2024.07.017](https://doi.org/10.1016/j.jid.2024.07.017)
* **Mumme HL**, Raikar SS, Bhasin SS et al. Single-cell RNA sequencing distinctly characterizes the wide heterogeneity in pediatric mixed phenotype acute leukemia. ***Genome Med*** 15, 83 (2023). [https://doi.org/10.1186/s13073-023-01241-z](https://doi.org/10.1186/s13073-023-01241-z)
* **Mumme H**, Thomas BE, Bhasin SS et al. Single-cell analysis reveals altered tumor microenvironments of relapse- and remission-associated pediatric acute myeloid leukemia. ***Nat Commun*** 14, 6209 (2023).
[https://doi.org/10.1038/s41467-023-41994-0](https://doi.org/10.1038/s41467-023-41994-0)
* Bhasin SS, Thomas BE, Summers RJ, Sarkar D, **Mumme H** et al. Pediatric T-cell acute lymphoblastic leukemia blast signature and MRD associated immune environment changes defined by single cell transcriptomics analysis. ***Sci Rep*** 13, 12556 (2023). [https://doi.org/10.1038/s41598-023-39152-z](https://doi.org/10.1038/s41598-023-39152-z)
* Dwivedi B, **Mumme H**, Satpathy S et al. Survival Genie, a web platform for survival analysis across pediatric and adult cancers. ***Sci Rep*** 12, 3069 (2022). [https://doi.org/10.1038/s41598-022-06841-0](https://doi.org/10.1038/s41598-022-06841-0)
* Theocharidis G, Thomas BE, Sarkar D, **Mumme HL** et al. Single cell transcriptomic landscape of diabetic foot ulcers. ***Nat Commun*** 13, 181 (2022). [https://doi.org/10.1038/s41467-021-27801-8](https://doi.org/10.1038/s41467-021-27801-8)
* Andueza A, Kumar S, Kim J, Kang DW, **Mumme HL** et al. Endothelial Reprogramming by Disturbed Flow Revealed by Single-Cell RNA and Chromatin Accessibility Study. ***Cell Rep*** 33, 11 (2020). [https://doi.org/10.1016/j.celrep.2020.108491](https://doi.org/10.1016/j.celrep.2020.108491)
* Mankoul-Mansour MM, El-Beyrouthy JB, **Mumme HL**, Freeman, EC. Photopolymerized microdomains in both lipid leaflets establish diffusive transport pathways across biomimetic membranes. ***Soft Matter*** 15, 43
(2019). [https://doi.org/10.1039/C9SM01658A](https://doi.org/10.1039/C9SM01658A)

PRESENTATIONS
======
* Poster Presentation, American Society of Hematology, Dec 2024. "CAR-Machine: CAR-T Cell Target Identification Platform for Pediatric Acute Leukemia and Implementation to Identify Novel Targets for Pediatric AML", [https://doi.org/10.1182/blood-2024-210175](https://doi.org/10.1182/blood-2024-210175)
* **Oral Presentation**, Aflac Cancer and Blood Disorders Center [Advances in Research](https://www.pedsresearch.org/research/centers/aflac/overview/aflac-research-conferences/air-aflac-research-conference), Nov 2024. "CAR-Machine: CAR T-Cell Target Identification Platform For Pediatric Acute Leukemia"
* Poster Presentation, [Atlanta Workshop on Single-cell Omics](https://research.gatech.edu/data/events/awsom), April 2024. "CAR-Machine: Resource for Single-Cell Guided CAR-T Target Identification and Testing for Hematological Childhood Cancers"
* Poster Presentation, [Atlanta Workshop on Single-cell Omics](https://research.gatech.edu/data/events/awsom), April 2024. "Single-cell Profile of Prostate Cancer PBMCs Before and After Androgen Deprivation Therapy"
* Poster Presentation, [St. Jude National Graduate Student Symposium](https://www.stjude.org/education-training/predoctoral-training/graduate-students/national-graduate-student-symposium-ngss.html), March 2024. "Single Cell RNA Sequencing Driven Characterization of Pediatric Mixed Phenotype Acute Leukemia"
* Poster Presentation, [Atlanta Workshop on Single-cell Omics](https://research.gatech.edu/data/events/awsom), April 2023. "Single Cell RNA Sequencing Driven Characterization of Pediatric Mixed Phenotype Acute Leukemia"
* **Oral Presentation**, American Society of Hematology, Dec 2022. "A Single Cell Atlas and Interactive Web-Resource of Pediatric Cancers and Healthy Bone Marrow", [https://doi.org/10.1182/blood-2022-169153](https://doi.org/10.1182/blood-2022-169153)
* **Oral Presentation**, Atlanta Workshop on Single-cell Omics, April 2022. "Pediatric Cancer Cell Atlas"
* Poster Presentation, American Society of Hematology, Dec 2021. "Pediatric Single Cell Cancer Atlas: An Integrative Web-Based Resource for Single Cell Transcriptome Data from Pediatric Leukemias",
[https://doi.org/10.1182/blood-2021-154282](https://doi.org/10.1182/blood-2021-154282)
* Poster Presentation, American Society of Hematology, Dec 2021. "Single Cell RNA Sequencing Driven Characterization of Rare B/Myeloid and T/Myeloid Mixed Phenotype Acute Leukemia",
[https://doi.org/10.1182/blood-2021-153776](https://doi.org/10.1182/blood-2021-153776)
  
TEACHING AND ACTIVITIES
======
* Emory CS 170 - *Introduction to Computer Science I* (***Java***), Graduate TA (Fall 2021)
* Atlanta Workshop on Single-cell Omics, Planning Committee (April 2022)
* Emory CS 170 - *Introduction to Computer Science I* (***Java***), Graduate TA (Fall 2022)
* Emory CS 470 - *Data Mining* (**Java/Python**), Graduate TA (Spring 2023)
* Emory QTM 150 - *Introduction to Statistical Computing I* (**R**), Graduate TA (Fall 2024)

WORKSHOPS
======
* NCI [Human Tumor Atlas Network (HTAN) Data Jamboree](https://humantumoratlas.org/jamboree) (December 2023)
  * download, process, and map immune infiltration of HTAN spatial transcriptomics (ST) data (using scRNA-seq as reference for immune signatures)
  * gained experience with analyzing 10X Visium data and pancreatic ductal adenocarcinoma (PDAC)
  * HTAN Immune Infiltration [Github](https://github.com/NCI-HTAN-Jamborees/Immune-infiltration) with our code and results

AWARDS
======
* Best Poster, [Atlanta Workshop on Single-cell Omics](https://research.gatech.edu/data/events/awsom) (April 2023)

SKILLS
======
**Programming**: Python (Matplotlib, Seaborn, Pandas, scikit-learn, PyTorch, plotly, dash), R (dplyr, tidyr, ggplot2, shiny), Bash, SQL, Matlab, MarkDown, Java, LaTeX, Git

**Omics Analysis**: scRNA-seq, scATAC-seq, CITE-seq, bulk RNA-seq, Spatial Transcriptomics (Visium), lncRNA

**Bioinformatics**: Seurat, Scanpy, batch correction, trajectory, RNA velocity, doublet identification, scvi-tools, GSEA, cellular communication, gene regulatory networks, cellranger, limma, STAR

**Communication**: Oral and poster presentations at scientific conferences, presenting projects to analytical team and clinical collaborators.

**AI**: Neural Networks, Regression, VAE, SVM, Random Forest, Transformers

-------------------

[Download a pdf of my CV.](../files/cv_hopemumme.pdf)

