---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

----------------

EDUCATION
======
* B.Sc. in Biomedical Engineering, Georgia Institute of Technology - Dec 2020
* Ph.D in Computer Science and Informatics - May 2025 (Expected)

RESEARCH
======
**Graduate Researcher, Bhasin Lab** - *Emory University (2021-)*
* Performed pediatric Pan-Leukemia transcriptomic analysis to identify 9 genes dysregulated in leukemia and over-expressed at diagnosis in AML, ALL, and MPAL
* Developed [PedSCAtlas](https://bhasinlab.bmi.emory.edu/PediatricSCAtlas/), first interactive tool for analyzing compiled datasets of pediatric single-cell RNA-seq cancer datasets for pediatric cancer subtypes
* Performed single-cell RNA-seq analysis of pediatric mixed phenotype acute leukemia (MPAL), and comparison to other acute leukemia subtypes
* Validated our groupâ€™s 7-gene signature for identification of AML malignant blast cells using external single-cell and bulk RNA-seq datasets
* Assisted with various bioinformatics projects in the lab, such as the development and maintenance of the web resource Survival Genie, single-cell analysis of T-cell acute lymphoid leukemia immune microenvironment, and the single-cell analysis of diabetic foot ulcer healing

**Undergraduate Researcher, Jo Lab** - *Georgia Institute of Technology (2019-2020)*
* Performed scRNA-seq, scATAC-seq, and lncRNA analysis to study endothelial cell reprogramming in the carotid artery

**Data Analysis Internship** - *Open Medicine Institute (2018)*
* Developed a single-cell RNA sequencing protocol for their patient data
  
PUBLICATIONS
======
* **Mumme HL**, Raikar SS, Bhasin SS et al. Single-cell RNA sequencing distinctly characterizes the wide heterogeneity in pediatric mixed phenotype acute leukemia. ***Genome Med*** 15, 83 (2023). [https://doi.org/10.1186/s13073-023-01241-z](https://doi.org/10.1186/s13073-023-01241-z)
* **Mumme H**, Thomas BE, Bhasin SS et al. Single-cell analysis reveals altered tumor microenvironments of relapse- and remission-associated pediatric acute myeloid leukemia. ***Nat Commun*** 14, 6209 (2023).
[https://doi.org/10.1038/s41467-023-41994-0](https://doi.org/10.1038/s41467-023-41994-0)
* Bhasin SS, Thomas BE, Summers RJ, Sarkar D, **Mumme H** et al. Pediatric T-cell acute lymphoblastic leukemia blast signature and MRD associated immune environment changes defined by single cell transcriptomics analysis. ***Sci Rep*** 13, 12556 (2023). [https://doi.org/10.1038/s41598-023-39152-z](https://doi.org/10.1038/s41598-023-39152-z)
* Dwivedi B, **Mumme H**, Satpathy S et al. Survival Genie, a web platform for survival analysis across pediatric and adult cancers. ***Sci Rep*** 12, 3069 (2022). [https://doi.org/10.1038/s41598-022-06841-0](https://doi.org/10.1038/s41598-022-06841-0)
* Theocharidis G, Thomas BE, Sarkar D, **Mumme HL** et al. Single cell transcriptomic landscape of diabetic foot ulcers. ***Nat Commun*** 13, 181 (2022). [https://doi.org/10.1038/s41467-021-27801-8](https://doi.org/10.1038/s41467-021-27801-8)
* Andueza A, Kumar S, Kim J, Kang DW, **Mumme HL** et al. Endothelial Reprogramming by Disturbed Flow Revealed by Single-Cell RNA and Chromatin Accessibility Study. ***Cell Rep*** 33, 11 (2020). [https://doi.org/10.1016/j.celrep.2020.108491](https://doi.org/10.1016/j.celrep.2020.108491)
* Mankoul-Mansour MM, El-Beyrouthy JB, **Mumme HL**, Freeman, EC. Photopolymerized microdomains in both lipid leaflets establish diffusive transport pathways across biomimetic membranes. ***Soft Matter*** 15, 43
(2019). [https://doi.org/10.1039/C9SM01658A](https://doi.org/10.1039/C9SM01658A)

PRESENTATIONS
======
* Poster Presentation, [Atlanta Workshop on Single-cell Omics](https://research.gatech.edu/data/events/awsom), April 2023. "Single Cell RNA Sequencing Driven Characterization of Pediatric Mixed Phenotype Acute Leukemia"
* Oral Presentation, American Society of Hematology, Dec 2022. "A Single Cell Atlas and Interactive Web-Resource of Pediatric Cancers and Healthy Bone Marrow", [https://doi.org/10.1182/blood-2022-169153](https://doi.org/10.1182/blood-2022-169153)
* Oral Presentation, Atlanta Workshop on Single-cell Omics, April 2022. "Pediatric Cancer Cell Atlas"
* Poster Presentation, American Society of Hematology, Dec 2021. "Pediatric Single Cell Cancer Atlas: An Integrative Web-Based Resource for Single Cell Transcriptome Data from Pediatric Leukemias",
[https://doi.org/10.1182/blood-2021-154282](https://doi.org/10.1182/blood-2021-154282)
* Poster Presentation, American Society of Hematology, Dec 2021. "Single Cell RNA Sequencing Driven Characterization of Rare B/Myeloid and T/Myeloid Mixed Phenotype Acute Leukemia",
[https://doi.org/10.1182/blood-2021-153776](https://doi.org/10.1182/blood-2021-153776)
  
TEACHING AND ACTIVITIES
======
* Emory CS 170, Graduate TA (Fall 2021)
* Atlanta Workshop on Single-cell Omics, Planning Committee (April 2022)
* Emory CS 170, Graduate TA (Fall 2022)
* Emory CS 470, Graduate TA (Spring 2023)

AWARDS
======
* Best Poster, [Atlanta Workshop on Single-cell Omics](https://research.gatech.edu/data/events/awsom) (April 2023)

SKILLS
======
**Programming**: R, Python, Latex, Matlab, MarkDown, Java, Bash

**Omics Analysis**: scRNA-seq, scATAC-seq, CITE-seq, bulk RNA-seq

**AI**: scikit-learn, tensorflow, pytorch

**Web Tool Dev**: R shiny, Python dash, HTML

**Other**: Github, Microsoft Office

======

<a href="files/cv_hopemumme_Oct23.pdf" download="hopemumme_CV.pdf">Download a pdf of my CV</a>
